client/components/notice/test/index.js

```diff
@@ -1,59 +1,62 @@
-import { assert } from 'chai';
-import { shallow } from 'enzyme';
+/**
+ * @jest-environment jsdom
+ */
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
 import { Notice } from '../index';
 
 describe( 'Notice', () => {
 	const translate = ( string ) => string;
 
 	test( 'should output the component', () => {
-		const wrapper = shallow( <Notice translate={ translate } /> );
-		assert.isOk( wrapper.find( '.notice' ).length );
+		const { container } = render( <Notice translate={ translate } /> );
+		expect( container.firstChild ).toHaveClass( 'notice' );
```

Comment:
For most of the tests in this file, should we query with a selector instead of using `container.firstChild`? `firstChild` feels more based on the DOM structure rather than what the user sees 🤔

Reply:
`container.firstChild` is actually the root element, but I see why it can be confusing.

I've used the opportunity to refactor a bit, and add a `role` and `aria-label` attributes. That way the appearing notice will be read by screen readers, "Notice: Settings saved" or "Notice: Something went wrong". 

So we now use `queryByRole` to retrieve the notice element.

What do you think?

Reply:
Love it, that makes a lot of sense to me!